client/src/pages/CreateJob.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+
+
+const CreateJob = () => {
```

Comment:
Can you also add the css file for this?

Reply:
In the CSS, add different colour to the button, padding, and margin.

Reply:
I have added tailwind, or what do you mean is for every page/component add a css file?

Reply:
I meant to add CreateJob.css or something like that.

Reply:
This one is answered later